aries-site/src/pages/components/skeleton.mdx

```diff
@@ -0,0 +1,42 @@
+import { Notification } from 'grommet';
+import { Example } from '../../layouts';
+import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
+
+<Example
+  componentName="Skeleton"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js"
+  docs="https://v2.grommet.io/skeleton?theme=hpe#props"
+  width="large"
+>
+  <DashBoardSkeletonExample />
+</Example>
+
+### When to use
+Skeletons are used on all pages to provide a visual layout of the content being loaded.  They help users perceive a loading period as being shorter in duration.
+
+### When not to use
+Skeletons should not be used if:
+- The data or elements being loaded are unknown.
+- The user is awaiting feedback for in-context actions such as submitting a form or fetching search results.
+  - The Spinner component may be more appropriate.
+- The loading process takes more than a few minutes to load.
+  - Pair with a message or notification to provide the user with better visibility of a system’s status.
+
+### Anatomy
+
+<Example plain>
+  <SkeletonAnatomy />
+</Example>
+
+
+Skeletons come in three different shapes - circular, rounded, and rectangular. The shape, size, and elevation of a Skeleton are all dependent on the UI elements that are loading. 
+
+Some common components that may be skeletonized are:
+- text, paragraph, heading
+- avatar, image
+- button,
+- card, table, pageheader, footer, header
+
+Grommet has added `skeleton` property in order to make it easy to switch between a skeleton state and data-rich state for a variety of components.
+The components that currently support skeleton are:
+- box, button, heading, paragraph, text
```

Comment:
Should we capitalize these since we are referring directly to the Grommet components?

Reply:
yes I can

Reply:
done